resources/i18n/en.json

```diff
@@ -24,8 +26,12 @@
       "OriginalPassword": "Original password",
       "NewPassword": "New password",
       "NewPasswordAgain": "New password (again)",
+      "InputOriginalPassword": "Please Input Original Password",
       "InvalidPasswordMessage": "At least 1 alphabet, 1 number and 1 special character is required with at least 8 chars.",
+      "NewPasswordMismatch": "Two new passwords do not match.",
       "PasswordMismatch": "Password mismatch",
+      "PasswordUpdated": "Password is successfully updated.",
+      "UsernameUpdated": "User full name is successfully updated.",
```

Comment:
`username` and `full_name` are completely different and [distinguished in the DB-level](https://github.com/lablup/backend.ai-manager/blob/master/src/ai/backend/manager/models/user.py#L93-L98). So, I prefer to distinguish them in the Console as well.